Updated contributing Guidelines (#378)
Direwolf20-MC/BuildingGadgets: CONTRIBUTION.md

* If testing was performed, what tests and how extensive they have been

## Code Guidelines

> In general your code should be well-structured and easy to read. we'd like to be the first to acknowledge that our code base isn't perfect and in some areas it's really not perfect! But we are trying to improve upon the code-base. Unless your PR is directly related to code cleaning then your PR should not contain any formatting changes to unchanged code blocks as this simply makes reviewing them unnecessarily complex.

Reviewer: My English teachers would have killed you for starting a sentence with _but_ XD... 
Also as a non-native speaker I actually had trouble understanding the code cleaning sentence - I'm unsure about the commas though... I never understood english punctuation...
```suggestion
In general your code should be well-structured and easy to read. we'd like to be the first to acknowledge that our code base isn't perfect and in some areas it's really unreadable! We are trying to improve upon the code-base. 
- Your PR should not contain any formatting changes to unchanged code blocks, unless it is directly related to code cleanup. This makes reviewing them unnecessarily complex.
```

Author: But is a weird one. It's not meant to be used at the start of a sentence if the sentence break has been added for the word but. Yeah, English grammar is crazy
